src/tests/unit/tests/DetailsView/components/start-over-component-factory.test.tsx

```diff
@@ -100,12 +103,22 @@ describe('StartOverComponentPropsFactory', () => {
         expect(rendered).toBeNull();
     });
 
-    test('getStartOverComponentPropsForFastPass, CardsUI is true, scanResults is not null, component matches snapshot', () => {
+    test('getStartOverComponentPropsForFastPass, CardsUI is true, scanResults is not null, scanning is false, component matches snapshot', () => {
         setScanResult();
         setCardsUiFlag(true);
         const props = getProps(false);
         const rendered = getStartOverComponentForFastPass(props);
 
         expect(rendered).toMatchSnapshot();
     });
+
+    test('getStartOverComponentPropsForFastPass, CardsUI is true, scanResults is not null, scanning is true, component matches snapshot', () => {
+        setScanResult();
+        setCardsUiFlag(true);
+        scanning = 'true';
```

Comment:
Just a heads up we don't usually set scanning to "true"; the string is usually a string identifier for a test.